ts-tests/tests/test-eth-balance.ts

```diff
@@ -145,7 +219,7 @@ async function get_assets(api: ApiPromise, alice: KeyringPair) {
   console.log(`Linked Ethereum balances of Alice are: ${assetsBalances.toString()}`);
   
   // TODO fetch real time balance and compare it here
-  expect(assetsBalances.toString()).to.equal(`[0,"0x000000000000000006f05b59d3b20000"]`);
+  expect(assetsBalances.toString()).to.equal(`[0,"0x00000000000000004563918244f40000"]`);
```

Comment:
How do you know that the ocw has returned

Reply:
we call `await asset_claim(api, alice);` before this function is called. Inside asset_clain, we wait until the `result.status.isFinalized` is returned.